sys-boot/refind/refind-0.14.0.2-r1.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs secureboot
+
+DESCRIPTION="The UEFI Boot Manager by Rod Smith"
+HOMEPAGE="https://www.rodsbooks.com/refind/"
+SRC_URI="mirror://sourceforge/project/${PN}/${PV}/${PN}-src-${PV}.tar.gz"
+
+LICENSE="BSD GPL-2 GPL-3 FDL-1.3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+FS_USE="btrfs +ext2 +ext4 hfs +iso9660 ntfs reiserfs"
+IUSE="${FS_USE} custom-cflags doc"
+
+DEPEND="sys-boot/gnu-efi"
+
+# for ld.bfd and objcopy
+BDEPEND="sys-devel/binutils"
+
+DOCS=( README.txt NEWS.txt )
+
+PATCHES=( "${FILESDIR}"/${P}-clang.patch )
+
+checktools() {
+	if [[ ${MERGE_TYPE} != "binary" ]]; then
+		# 832018
+		export LD="$(tc-getLD)"
+		tc-ld-force-bfd
+		# tc-ld-force-bfd does not work in all cases, so try to also fix LD
+		LD="${LD/.lld/.bfd}"
+		tc-ld-is-lld "${LD}" && die "Linking with lld produces broken executables and may lead to unbootable system"
+
+		# 732256
+		# llvm-objcopy does not support EFI target, try to use binutils objcopy or fail
+		export OBJCOPY="$(tc-getOBJCOPY)"
+		OBJCOPY="${OBJCOPY/llvm-/}"
+		LANG=C LC_ALL=C "${OBJCOPY}" --help | grep -q '\<pei-' || die "${OBJCOPY} (objcopy) does not support EFI target"
+	fi
+}
+
+pkg_pretend() {
+	if use custom-cflags; then
+		ewarn
+		ewarn "You have enabled building with USE=custom-cflags. Be aware that"
```

Comment:
I wonder why we're bothering then. Do we really want this flag?

Reply:
Well, we had a long discussion some years ago. Gentoo users really like to put their own CFLAGS, so we should allow them to do so. But compiling EFI is very sensitive to these flags, and compilation will have great chances to fail if not using the default flags as set upstream. That’s why we used the custom-cflags and output a warning.

Reply:
Nowadays, we tend not to indulge people if it's really silly to. This is one example of that where it gives no real value at all. 

What would be nice is to do `strip-flags` and just append it / prepend it to the upstream ones, unconditionally.

Reply:
I would rather drop the custom flags actually.  I tried to build it with some custom flags and almost always I obtained an unbootable system (if it compiled at all).  I didn't remove it yet because it is masked and I wanted a second opinion.

Reply:
Well, I feel there is a consensus for removing it… :)